staking/effective/eligible.go

```diff
@@ -7,28 +7,43 @@ type Eligibility byte
 const (
 	// Nil is a default state that represents a no-op
 	Nil Eligibility = iota
-	// Active means allowed in epos auction
-	Active
-	// Inactive means validator did not sign enough over 66%
+	// FirstTimeCandidate ..
+	FirstTimeCandidate
```

Comment:
I think these new statuses are too much and may cause more potential mistakes in coding. I don't see the extra benefit outweigh the potential issues.

Reply:
Ok, then I think we can remove firstTimeCandidate, it's not necessary. And basically it's two bit, candidate/not, incommittee/not.

We can do 

1. Candidate
2. NotCandidate
3. ElectedAndSigning
4. ElectedAndFellBelowThreshold